### PR TITLE
Create placeholder pages for administrator and coordinator roles

### LINKED ISSUE #419: Create placeholder pages for administrator and coordinator roles
Create placeholder pages for administrator and coordinator functionality like election, user and workstation management. Change the navigation bar to add all possible roles for that page in the top right.

### LINKED ISSUE #420: Move end of session / PDF creation to coordinator role
Move end of session / PDF creation from the data entry role to coordinator role.

* 'Data entry completed' alert should be moved to `/elections/{election_id}/status`
* 'Create report' page should be moved to `/elections/{election_id}/report`
* Add d2d test?

### PR DESCRIPTION
**Pull request by Lionqueen94**
Closes #419 
Closes #420

## Description
- [x] Added placeholder pages for:
  - `/elections/{election_id}/status`
    - coordinator role: data entry status
  - `/users` 
    - administrator+coordinator roles: configure users
  - `/workstations` 
    - administrator role: configure workstations
  - `/logs` 
    - administrator+coordinator roles: view logs
- [x] 'Create report' page is moved to `/elections/{election_id}/report`
- [x] Updated `/dev` to organize the links based on roles
- [x] Added possible roles for each page based on hash in URL (for now, will show the user role once user accounts are implemented)
- [x] Fixed styling of NavBar and made sure the NavBar is correctly shown on each page
- [x] Improved styling of Alert content (max-length for the text for example)
- [x] Renamed folder `data_entry/PollingStation` to `data_entry/polling_station`
- [x] Moved form pages from `data_entry/polling_station` to `data_entry/page`
- [x] 'Data entry completed' Alert is added to `/elections/{election_id}/status` and tested
- [x] 'Data entry completed' Alert is updated on `/elections/{election_id}/data-entry`

### CODE REVIEW

**Comment by jorisleker** [comment]
@Lionqueen94 If I use the urls as provided in the description of the PR, I see 'invoerder' on the top right. I noticed that you use an url-anchor to provide the role. This is a temporary solution I guess?

**Review by jorisleker — DISMISSED** [review]

**Comment by Lionqueen94** [comment]
> @Lionqueen94 If I use the urls as provided in the description of the PR, I see 'invoerder' on the top right. I noticed that you use an url-anchor to provide the role. This is a temporary solution I guess?

Yes for sure, once we have implemented user accounts, this will show the role of the user. For now I made sure to add the right hashes coming from the `/dev` homepage and to add the right hashes to the tabs you can click when you are administrator/coordinator

File: frontend/app/module/DevHomePage.tsx
[the remark this answers is not in the document]
Comment (Lionqueen94) [reply frontend/app/module/DevHomePage.tsx:1]: Done!

**Review by jorisleker — DISMISSED** [review]

File: frontend/app/module/election/page/ElectionStatusPage.test.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+const renderElectionStatusPage = () =>
+  render(
+    <ElectionProvider electionId={1}>
+      <ElectionStatusProvider electionId={1}>
+        <ElectionStatusPage />
+      </ElectionStatusProvider>
+    </ElectionProvider>,
+  );
+
+describe("DataEntryHomePage", () => {
```
Comment (oliver3) [inline frontend/app/module/election/page/ElectionStatusPage.test.tsx:18]:
```suggestion
describe("ElectionStatusPage", () => {
```

File: frontend/app/routes.tsx [hunk 2]
```diff
@@ -1,55 +1,70 @@
...
+      <Route path=":electionId" element={<ElectionLayout />} errorElement={<NotFound />}>
+        <Route index element={<ElectionHomePage />} />
         <Route
-          path="finalise"
-          element={__API_MSW__ ? <NotAvailableInMock title="Invoerfase afronden - Abacus" /> : <FinaliseElectionPage />}
+          path="report"
+          element={__API_MSW__ ? <NotAvailableInMock title="Invoerfase afronden - Abacus" /> : <ElectionReportPage />}
         />
+        <Route path="status" element={<ElectionStatusPage />} />
+        <Route path="data-entry" element={null}>
```
Comment (oliver3) [inline frontend/app/routes.tsx:49]: Maybe remove this Route and add `data-entry` to the two nested routes below?

File: frontend/app/module/election/page/ElectionHomePage.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import { MockTest } from "app/component/MockTest";
+import { NavBar } from "app/component/navbar/NavBar";
+
+import { useElection } from "@kiesraad/api";
+
+export function ElectionHomePage() {
+  const { election } = useElection();
+
+  return (
+    <>
```
Comment (oliver3) [inline frontend/app/module/election/page/ElectionHomePage.tsx:12]: Wondering if this page should have a `PageTitle`..

File: frontend/app/module/election/page/ElectionStatusPage.test.tsx (hunk 1 above)
Comment (Lionqueen94) [reply frontend/app/module/election/page/ElectionStatusPage.test.tsx]: Fixed in 623a7a1ad6c1fea13ddaba577f23e2f61d29b1a4

File: frontend/app/module/election/page/ElectionHomePage.tsx (hunk 3 above)
Comment (Lionqueen94) [reply frontend/app/module/election/page/ElectionHomePage.tsx:13]: Added in 623a7a1ad6c1fea13ddaba577f23e2f61d29b1a4

**Review by oliver3 — APPROVED** [review]

**Review by lkleuver — APPROVED** [review]

**Pull request merged**